aidial_adapter_openai/app.py

```diff
@@ -67,11 +71,25 @@ async def chat_completion(deployment_id: str, request: Request):
     api_type, api_key = await get_credentials(request)
 
     upstream_endpoint = request.headers["X-UPSTREAM-ENDPOINT"]
+    api_version = get_api_version(request)
+
+    if api_version == "":
```

Comment:
Btw, this check could be moved into get_api_version, since it always needed after get_api_version call.